pkg/retry/retryer.go

```diff
@@ -103,9 +180,18 @@ type Container struct {
 }
 
 type cbContainer struct {
+	cbSuite *circuitbreak.CBSuite
```

Comment:
both have cbSuite and cbCtl/cbPanel, add comment to describe is better

Reply:
Added some comments on this.